File: LeetCode/Easy/0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int temp = 0;

        for(int i = 0; i < nums.length; ++i) {
            if(nums[i] == 0) {
                max = Math.max(max, temp);
                temp = 0;
            } else {
                temp++;
            }
        }
        return Math.max(max, temp);
    }
}

File: LeetCode/Easy/0485-max-consecutive-ones/0485-max-consecutive-ones.kt
class Solution {
    fun findMaxConsecutiveOnes(nums: IntArray): Int {
        var max: Int = 0
        var temp: Int = 0

        for (num in nums) {
            if(num == 0) {
                max = Math.max(max, temp)
                temp = 0
            } else {
                temp++
            }
        }
        return Math.max(max, temp)
    }
}
